extensions/amp-font/0.1/amp-font.js

```diff
@@ -124,7 +128,7 @@ export class AmpFont extends AMP.BaseElement {
       this.onFontLoadSuccess_();
     }).catch(error => {
       this.onFontLoadError_();
-      throw error;
+      log.warn(TAG_, error, this.element);
```

Comment:
@sriramkrish85 @erwinmombay is working on unifying the 2.

For now add console.warn